gcloud_dataproc/load_dataset.py

```diff
@@ -40,18 +40,18 @@ def init_command_line_args():
     p.add_argument("--stop-after-step", help="stop after this pipeline step", type=int)
     p.add_argument("--download-fam-file", help="download .fam file from seqr", action='store_true')
 
+
     p.add_argument("--use-temp-loading-nodes", action="store_true",
         help="If specified, temporary loading nodes will be created and added to the elasticsearch cluster")
+    p.add_argument("--k8s-cluster-name", help="Specifies the kubernetes cluster name that hosts elasticsearch (eg. 'gcloud-prod-es')."
+        "This name is currently also re-used as elasticsearch's internal cluster name which it uses to link up with other elasticsearch instances.", default=random_es_cluster_name)
     p.add_argument("--num-temp-loading-nodes", type=int,
         help="For use with --num-temp-loading-nodes. Number of temp loading nodes to create.", default=3)
 
     p.add_argument("--create-persistent-es-nodes", action="store_true",
         help="If specified, a persistent ES cluster will be created before loading data or creating temp loading nodes."
         " This is unnecessary if an elasticsearch cluster already exists.")
-    p.add_argument("--k8s-cluster-name", help="Specifies the kubernetes cluster name that hosts elasticsearch.", default=random_es_cluster_name)
-
-    group = p.add_mutually_exclusive_group()
-    group.add_argument("--num-persistent-nodes", type=int, help="For use with --num-persistent-nodes. Number of persistent data nodes to create.", default=3)
+    p.add_argument("--num-persistent-nodes", type=int, help="For use with --num-persistent-nodes. Number of persistent data nodes to create.", default=3)
```

Comment:
I actually took this arg out because its not used at all for this, its only used in restore_disks_from_snapshots